edgraph/server.go

```diff
@@ -148,12 +149,12 @@ func GetGQLSchema(namespace uint64) (uid, graphQLSchema string, err error) {
 	resp, err := (&Server{}).QueryNoGrpc(ctx,
 		&api.Request{
 			Query: `
-			query {
```

Comment:
have we made a change here? If not, let's undo this.

Reply:
That is go fmt, I won't pick a fight with it. Either I disable it (and lose any benefit from using it) or I just let it do what it wants to do. Is there any default config to enforce go fmt not doing it?

Reply:
I am not sure why it is doing it for you. My vscode also runs go fmt and it is not updating this file for me. In any case, you still should be able to undo this change. (run `git checkout origin/main edgraph/server.go`

Reply:
This command undo everything(all commits, not just that part) and also bring new stuff. It doesn't work as you expected.

I have nothing fancy in my Go env. Just started it and done. Maybe it is the version?

```
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/root/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.20.1"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/root/dev/dgraph/go.mod"
GOWORK=""
CGO_CFLAGS="-O2 -g"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-O2 -g"
CGO_FFLAGS="-O2 -g"
CGO_LDFLAGS="-O2 -g"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build463741918=/tmp/go-build -gno-record-gcc-switches"
```